xml/System.Numerics/BitOperations.xml

```diff
@@ -42,10 +42,20 @@
         <Parameter Name="value" Type="System.UInt32" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The mask.</param>
+        <summary>Counts the number of leading zero bits in an unsigned 32-bit integer mask.</summary>
+        <returns>The number of leading zero bits in a mask.</returns>
+        <remarks>
+          <format type="text/markdown">
+            <![CDATA[
+
+### Remarks
+
+Similar in behavior to the x86 instruction LZCNT.
```

Comment:
It would be nice if this documented the behavior of the edge case handling and indicated it used `LZCNT` on x86 instead...

Reply:
@tannergooding since you're the area owner, would you mind suggesting the correct text? Press Control + G to add a code suggestion for this line.

Reply:
```suggestion
Returns 32 when <paramref name="value" /> is 0.

This API utilizes System.Runtime.Intrinsics.X86.Lzcnt.LeadingZeroCount when System.Runtime.Intrinsics.X86.Lzcnt.IsSupported returns true.
```